dynawo/sources/Solvers/AlgebraicSolvers/DYNSolverKINEuler.cpp

```diff
@@ -73,6 +73,8 @@ SolverKINEuler::init(const shared_ptr<Model>& model, const std::string& linearSo
   int sizeY = model_->sizeY();
   if (sizeY == 0)
     throw DYNError(Error::SUNDIALS_ERROR, SolverEmptyYVector);
+  if (sizeY != model_->sizeF())
```

Comment:
else if?

Reply:
since the one above is throwing when true, it is equivalent. I can modify it if you prefer

Reply:
You are of course absolutely right ;)